13_sandboxes/test_case_generator.py

```diff
@@ -0,0 +1,274 @@
+# ---
```

Comment:
Would love feedback on high level structure before diving too much into the weeds - please see PR description for high level q's.